daemon/start_linux.go

```diff
@@ -1,30 +1,23 @@
 package daemon // import "github.com/docker/docker/daemon"
 
 import (
-	"context"
-	"fmt"
-
-	specs "github.com/opencontainers/runtime-spec/specs-go"
-
 	"github.com/docker/docker/container"
 	"github.com/docker/docker/errdefs"
-	"github.com/docker/docker/libcontainerd/types"
 	"github.com/docker/docker/oci"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
 )
 
 // initializeCreatedTask performs any initialization that needs to be done to
-// prepare a freshly-created task to be started.
-func (daemon *Daemon) initializeCreatedTask(ctx context.Context, tsk types.Task, container *container.Container, spec *specs.Spec) error {
+// prepare a freshly-created container to be started.
+func (daemon *Daemon) initializeCreatedContainer(container *container.Container, spec *specs.Spec) error {
```

Comment:
Wondering if we should keep the `context.Context` (assuming we may need it again at some point), although I guess we could add it back when that moment arrives.

Reply:
I'm happy to put it back ... but yes, my thinking was that as it's a non-exported function it can always be added back when needed.

Reply:
I'd like to keep the function name and signature unchanged as it makes for less code to review in both this PR and in the future PR which carries the next version of the network config code. (It's just noise to have to touch start_notlinux.go)

Reply:
@corhere you mean keeping the signature as it was before this PR? (so adjust the patch to not change the signature?)